### PR TITLE
Fix /var/mail right at container initialization rather than startup

### PR DESCRIPTION
**Pull request by kbeaugrand**

### CODE REVIEW

File: target/start-mailserver.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ DEFAULT_VARS["ENABLE_FETCHMAIL"]="${ENABLE_FETCHMAIL:="0"}"
 DEFAULT_VARS["ENABLE_LDAP"]="${ENABLE_LDAP:="0"}"
 DEFAULT_VARS["ENABLE_SASLAUTHD"]="${ENABLE_SASLAUTHD:="0"}"
 DEFAULT_VARS["SMTP_ONLY"]="${SMTP_ONLY:="0"}"
+DEFAULT_VARS["POSTFIX_MYDESTINATION"]="${POSTFIX_MYDESTINATION:=""}"
```
Comment (kbeaugrand) [inline target/start-mailserver.sh]: In my newest commits, I've changed this in order to have virtual domains.

Comment (tomav) [inline target/start-mailserver.sh]: We can discuss about your needs. Let me know.

**Comment by kbeaugrand** [comment]
I Closed the pull request in order to put you a new one that's working for me.

________________________________
De : Thomas VIAL <notifications@github.com>
Envoyé : dimanche 1 janvier 2017 15:08:31
À : tomav/docker-mailserver
Cc : Kevin BEAUGRAND; State change
Objet : Re: [tomav/docker-mailserver] Fix /var/mail right at container initialization rather than startup (#444)


@tomav commented on this pull request.

________________________________

In target/start-mailserver.sh<https://github.com/tomav/docker-mailserver/pull/444>:

> @@ -16,6 +16,7 @@ DEFAULT_VARS["ENABLE_FETCHMAIL"]="${ENABLE_FETCHMAIL:="0"}"
 DEFAULT_VARS["ENABLE_LDAP"]="${ENABLE_LDAP:="0"}"
 DEFAULT_VARS["ENABLE_SASLAUTHD"]="${ENABLE_SASLAUTHD:="0"}"
 DEFAULT_VARS["SMTP_ONLY"]="${SMTP_ONLY:="0"}"
+DEFAULT_VARS["POSTFIX_MYDESTINATION"]="${POSTFIX_MYDESTINATION:=""}"


We can discuss about your needs. Let me know.

-

File: target/start-mailserver.sh [hunk 2]
```diff
@@ -95,7 +97,7 @@ function register_functions() {
 
 	################### >> fix funcs
 
-	_register_fix_function "_fix_var_mail_permissions"
+#	_register_fix_function "_fix_var_mail_permissions"
```
Comment (kbeaugrand) [inline target/start-mailserver.sh]: The current function is changing the owner of all mailboxes to 5000:5000 (docker user). 
But while running the server, the mailboxes will be created with userid and group id (not with 5000:5000) with rights 7000. 
When the container restarts , the owner of mailboxes are changed to 5000 and mailboxes won't be accessible. 

Im trying to fix the owner of /var/mail when the container is initialized (in DokerFile) And letthe mailboxe owner as they have been created.

Comment (tomav) [inline target/start-mailserver.sh]: Where this uid `7000` comes from?

Comment (kbeaugrand) [inline target/start-mailserver.sh]: Sorry, not 7000 owner but 700 rights : 
```drwx------  8 ```

**Pull request closed without merging**